docs/sources/panels-visualizations/visualizations/datagrid/index.md

```diff
@@ -0,0 +1,88 @@
+---
+aliases:
+  - ../../features/panels/datagrid_panel/
+  - ../../reference/datagrid/
+  - ../../visualizations/datagrid/
+description: Learn about datagrid panel visualization features.
+keywords:
+  - grafana
+  - dashboard
+  - panels
+  - datagrid
+  - datagrid panel
+  - datagrid options
+menuTitle: Datagrid
+title: Datagrid
+weight: 350
+---
+
+# Datagrid
+
+{{% admonition type="note" %}}
+
+The Grafana Datagrid panel is Experimental. This feature is supported by the engineering team on a best-effort basis, and breaking changes may occur without notice prior to general availability.
+
+{{% /admonition %}}
+
+The Datagrid panel offers you the ability to create, edit, and fine-tune data within Grafana. As such, this panel can act as a data source for other panels
+inside a dashboard. Let us dive deeper into the flows and functionalities of this panel.
+
+![Datagrid panel](/media/docs/datagrid/screenshot-grafana-datagrid-panel.png)
+
+## Use cases
+
+The datagrid panel can be used to manipulate data queried from any data source, or from a dragged and dropped file. It can then be used as a simple tabular
+visualization, or the data can be modified - and even removed altogether - to create a blank slate. Any changes made to data source retrieved data triggers a confirmation modal that informs you about the action you are about to take. Editing the dataset changes the data source to use the inbuilt `-- Grafana --` data source, thus losing the old data source settings and related queries, while also copying the current dataset into the dashboards model. After confirming the action, all data modifications are saved to the dashboard model.
```

Comment:
Hey @imatwawana.

I agree we don't need to give info regarding 1 and 4. For 2, we're changing the datasources because a typical workflow would involve pulling some data from some datasource (e.g: mysql) and then the user might want to fine-tune the resulted data. In that case, the entire dataset will be saved as a snapshot which is a query type under the `-- Grafana --` datasource. This is to show that we no long query data from MySQL, but use locally stored data. For 3 I also agree that `replace` is better.

Reply:
Okay i've updated my edit to incorporate removing the stuff from 1 and 4 and hopefully accurately reflecting your explanation.

Reply:
Thank you! I just want to mention that it does not have to be MySQL neccesarily, but any datasource can be used to pull data and then editing it through the datagrid will copy and move that data to a Grafana DS Snapshot